### PR TITLE
[BACKLOG-36954] - Changed dialogs to responsive and removed fixed lengths

### PR DESCRIPTION
**Pull request by jsaraf**

### CODE REVIEW

**Comment by jsaraf** [comment]
Please review @pentaho/wcag

**Comment by buildguy** [comment]
### :x: Build failed in 1m 39s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by dcleao** [comment]
Replaces: https://github.com/pentaho/pentaho-platform/pull/5265

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 13m 9s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5268/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css [hunk 1]
```diff
@@ -1270,42 +1267,6 @@ div#customDropdownPopupMajor .gwt-MenuBar-vertical .gwt-MenuItem:hover:not(.gwt-
...
-  margin-bottom: 5px;
-}
-
-.schedulesPanel .pentaho-table th {
-  font-size: 12px;
-  border-spacing: 0;
-  color: #FFF;
-  padding: 8px 10px 10px 10px;
-  font-weight: normal;
-}
```
Comment (dcleao) [inline user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css:1287]: These should be kept here. See related comment: https://github.com/pentaho/pentaho-commons-gwt-modules/pull/877#discussion_r1152113875.

Comment (jsaraf) [reply user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css]: Added back.

**Review by dcleao — APPROVED** [review]

**Review by hv-vamsi — APPROVED** [review]

**Review by rmansoor — APPROVED** [review]

**Pull request merged**